build_projects/dotnet-host-build/DebTargets.cs

```diff
@@ -267,5 +267,10 @@ private static bool DebuildNotPresent()
         {
             return Cmd("/usr/bin/env", "debuild", "-h").Execute().ExitCode != 0;
         }
+
+        private static bool ShouldSkipBuildDebPackages()
+        {
+            return Environment.GetEnvironmentVariable("DOTNET_BUILD_SKIP_DEB_PACKAGING") == "1";
```

Comment:
@gkhanna79 can you take another look?  I added `--skip-deb-package-build` to build.sh as well which sets the environment variable.